components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -48,6 +49,7 @@ public class ActionManagementServiceImpl implements ActionManagementService {
     private static final CacheBackedActionMgtDAO CACHE_BACKED_DAO =
             new CacheBackedActionMgtDAO(new ActionManagementDAOImpl());
     private static final ActionValidator ACTION_VALIDATOR = new ActionValidator();
+    private static final ActionManagementAuditLoggerUtil auditLogger = new ActionManagementAuditLoggerUtil();
```

Comment:
Shall we remove 'Util' from class name

Reply:
This is updated with c81491d0d97828a7760ca1c4dc028f0ab9cd0b8b